src/lib/commandclass/ColorSwitchCC.ts

```diff
@@ -533,53 +380,40 @@ export class ColorSwitchCCReport extends ColorSwitchCC {
 		super(driver, options);
 
 		validatePayload(this.payload.length >= 2);
-		this._colorComponent = this.payload[0];
-		this._currentValue = this.payload[1];
+		this.colorComponent = this.payload[0];
+		this.currentValue = this.payload[1];
 
-		if (this.version >= 3) {
-			validatePayload(this.payload.length >= 4);
-			this._targetValue = this.payload[2];
-			this._duration = Duration.parseReport(this.payload[3]);
-		}
-
-		const colorKey = getColorKeyFromComponent(this._colorComponent);
-		if (!colorKey) {
-			return;
+		if (this.version >= 3 && this.payload.length >= 4) {
```

Comment:
@RoboPhred This is the reason your v1 deserialization test failed. You want to make sure that the CC is at least valid in V1 (validatePayload), but every version above that needs to treat the additional values as optional.